packages/app/src/server/service/search-delegator/elasticsearch.ts

```diff
@@ -303,7 +311,7 @@ class ElasticsearchDelegator implements SearchDelegator<Data> {
   }
 
   async createIndex(index) {
-    const body = require('^/resource/search/mappings.json');
+    const body = this.isElasticsearchV6 ? require('^/resource/search/mappings.json') : require('^/resource/search/mappings-es7.json');
```

Comment:
Please rename the current `mappings.json` to `mappings-es6.json` and leave `mappings-es7.json` as is.

Please fix in https://youtrack.weseek.co.jp/issue/GW-7608.